pydra/engine/helpers.py

```diff
@@ -158,13 +159,46 @@ def make_klass(spec):
     finally:
         # wait for the process to exit
         rc = await process.wait()
-    return rc, stdout, stderr
+    if strip:
+        return rc, stdout.rstrip(), stderr
+    else:
+        return rc, stdout, stderr
+
+
+def read_and_display(*cmd, strip=False, hide_display=False):
+    """Capture cmd's stdout
 
+    """
+    try:
+        process = sp.run(cmd, stdout=sp.PIPE, stderr=sp.PIPE)
```

Comment:
subprocess should take in environment settings, which should be settable at the shelltask level.